media/static-bundles.json

```
@@ -1004,6 +1004,13 @@
         "css/externalpages/pocket/components/_add.scss"
       ],
       "name": "pocket-add"
+    }, 
+    {
+      "files": [
+        "css/externalpages/pocket/style.scss",
+        "css/externalpages/pocket/components/_contact-info.scss"
```

This isn't a something introduced in this PR, but afaik in Sass it's a common convention to add a leading underscore to files that represent partials to be imported using `@import`. Would it be better to avoid that for these stylesheets since they represent whole pages? Maybe we could use:

```suggestion
        "css/externalpages/pocket/pages/contact-info.scss"
```